pubspec.yaml

```diff
@@ -40,7 +40,7 @@ dependencies:
   convert: ^3.1.1
   crypto: ^3.0.3
   device_info_plus: ^11.2.0
-  drift: ^2.5.0
+  drift: ^2.23.0
```

Comment:
Need help checking if podfiles need an update; deps: Upgrade drift to 2.23.0

Confirmed by running `tools/upgrade pod` at this commit, on a Mac, that no changes to `ios/Podfile.lock` or `macos/Podfile.lock` are needed.

Reply:
Thanks for confirming@